jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/ECSExplorerRootNode.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer.nodes
+
+import com.intellij.openapi.project.Project
+import software.amazon.awssdk.services.ecs.EcsClient
+import software.aws.toolkits.jetbrains.services.ecs.EcsParentNode
+import software.aws.toolkits.resources.message
+
+class ECSExplorerRootNode : AwsExplorerServiceNode {
```

Comment:
We typically do not upper-case acronyms, so `ECS` should be `Ecs`

Reply:
Corrected in new commit.